ds-card-reader/res/configuration.yaml

```diff
@@ -0,0 +1,80 @@
+# Copyright © 2023 Intel Corporation. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+
+MaxEventSize: 0
+Writable:
+  LogLevel: INFO
+  InsecureSecrets:
+    DB:
+      path: redisdb
+      Secrets:
+        username: ''
+        password: ''
+  Reading:
+    ReadingUnits: true
+Service:
+  HealthCheckInterval: 10s
+  Host: localhost
+  ServerBindAddr: ''
+  Port: 48098
+  StartupMsg: Card Reader device service started
+  MaxRequestSize: 0
+  RequestTimeout: 5s
+  CORSConfiguration:
+    EnableCORS: false
+    CORSAllowCredentials: false
+    CORSAllowedOrigin: 'https://localhost'
+    CORSAllowedMethods: 'GET, POST, PUT, PATCH, DELETE'
+    CORSAllowedHeaders: >-
+      Authorization, Accept, Accept-Language, Content-Language, Content-Type,
+      X-Correlation-ID
+    CORSExposeHeaders: >-
+      Cache-Control, Content-Language, Content-Length, Content-Type, Expires,
+      Last-Modified, Pragma, X-Correlation-ID
+    CORSMaxAge: 3600
+Registry:
+  Host: localhost
+  Port: 8500
+  Type: consul
+Clients:
+  core-data:
+    Protocol: http
+    Host: localhost
+    Port: 59880
+  core-metadata:
+    Protocol: http
+    Host: localhost
+    Port: 59881
```

Comment:
remove

Reply:
I think we still need them

Reply:
No, we don't. See config changes made in this PR
https://github.com/intel-retail/automated-vending/pull/145